web_src/less/_base.less

```diff
@@ -1116,6 +1116,7 @@ a.ui.card:hover,
 
 .ui.modal > .content {
   background: var(--color-body);
+  text-align: left !important;
```

Comment:
It's an historical problem as old as
* https://github.com/go-gitea/gitea/pull/16909/files#diff-5aac754faa1361a704b9e4ce1037521a44b8c04ac4ac09761561544c8651aa76R26-R27


And it also caused other problems:
* https://github.com/go-gitea/gitea/pull/15532#issuecomment-933603588

But I do not feel good to  pollute it globally again. But not a blocker since it was already messy for long time.

Reply:
The label editing modal and in my opinion other similar modals just look a bit messy without it, as shown in a screenshot in a previous comment.

It's not critical, I can leave out if you prefer not to do this.

Reply:
I think it's fine for me, actually I have a plan to remove all other polluted `text-align` in the future.